# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/cranelift/baldrapi.h
# Commit: 44b0c0bed8a6
# Full Hash: 44b0c0bed8a6b4debb950e7e1c69190883c65d60
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Atomic-rmw baseline operations. r=yury
#   
#   Atomic ADD, SUB, AND, OR, XOR, XCHG, CMPXCHG, as well as WAIT and
#   NOTIFY for the baseline compiler.
#   
# ==============================================================================

diff -r fb02656bb788 -r 44b0c0bed8a6 js/src/wasm/cranelift/baldrapi.h
--- a/js/src/wasm/cranelift/baldrapi.h	Mon Oct 18 09:58:15 2021 +0000
+++ b/js/src/wasm/cranelift/baldrapi.h	Mon Oct 18 09:58:15 2021 +0000
@@ -209,9 +209,9 @@
   TruncF64,
   PreBarrier,
   PostBarrier,
-  WaitI32,
-  WaitI64,
-  Wake,
+  WaitI32, /* WaitI32M32 */
+  WaitI64, /* WaitI64M32 */
+  Wake,    /* WakeM32 */
   Limit
 };
 
